indico/modules/events/abstracts/controllers/management.py

```diff
@@ -103,11 +104,47 @@ def _process(self):
             contrib_session = abstract_data.pop('contrib_session', None)
             merge_persons = abstract_data.pop('merge_persons', None)
             judge_abstract(self.abstract, abstract_data, judgment=judgment, contrib_session=contrib_session,
-                           merge_persons=merge_persons, send_notification=send_notification, )
+                           merge_persons=merge_persons, send_notification=send_notification)
         return WPManageAbstracts.render_template('management/abstract.html', self._conf, abstract=self.abstract,
                                                  form=form)
 
 
+class RHManageBulkAbstractJudgment(RHManageAbstractsActionsBase):
+    """Perform bulk judgment operations on selected abstracts"""
+
+    def _process(self):
+        abstract_ids = request.form.getlist('abstract_id')
+        form = AbstractsJudgmentForm(event=self.event_new, abstract_id=[a.id for a in self.abstracts])
+        action = request.values.get('action')
+        # The following could also be achived with:
+        # form.judgment.data = action if action in {'accept', 'reject', 'merge'} else None
```

Comment:
Thing is that `AbstractAction.xxxx.name` is the same as the argument I pass, that's why the above line works fine.